{
  if (isValid()) {
    try {
      GitOperation<Object> gitop=new GitOperation<Object>(){
        public Object call(        Git git,        GitContext context) throws Exception {
          return null;
        }
      }
;
      executeRead(gitop,true);
    }
 catch (    Exception e) {
      LOGGER.warn("Failed to perform a pull " + e,e);
    }
  }
}
